site/docs/pages/transaction/types.mdx

```diff
@@ -27,7 +27,7 @@ type TransactionError = {
 
 ```ts
 type TransactionReact = {
-  address: Address; // The wallet address involved in the transaction.
+  address?: Address; // The wallet address involved in the transaction (defaults to connected account if not provided).
```

Comment:
And in the end, we decided to remove this completelly, right?

Reply:
yep just updated